app/src/organisms/ProtocolSetupLiquids/LiquidDetails.tsx

```diff
@@ -0,0 +1,141 @@
+  white-space: break-spaces;
+  text-overflow: ${WRAP};
+  &:first-child {
+    border-top-left-radius: ${BORDERS.size_three};
+    border-bottom-left-radius: ${BORDERS.size_three};
+    width: 20%;
+  }
+  &:last-child {
+    border-top-right-radius: ${BORDERS.size_three};
+    border-bottom-right-radius: ${BORDERS.size_three};
+  }
+`
+
+interface LiquidDetailsProps {
+  liquid: ParsedLiquid
+  labwareByLiquidId: LabwareByLiquidId
+  runId: string
+  commands?: RunTimeCommand[]
+}
+
+export function LiquidDetails(props: LiquidDetailsProps): JSX.Element {
+  const { liquid, labwareByLiquidId, runId, commands } = props
+  const { t } = useTranslation('protocol_setup')
+  const [labwareIdModal, setLabwareId] = React.useState<string | null>(null)
+  return (
+    <Flex marginTop={SPACING.spacing5}>
+      {labwareIdModal != null && (
+        <LiquidsLabwareDetailsModal
+          labwareId={labwareIdModal}
+          liquidId={liquid.id}
+          runId={runId}
+          closeModal={() => setLabwareId(null)}
+        />
+      )}
+      <Table>
+        <thead>
+          <tr>
+            <TableHeader>{t('location')}</TableHeader>
+            <TableHeader>{t('labware_name')}</TableHeader>
+            <TableHeader>{t('volume')}</TableHeader>
+          </tr>
+        </thead>
+        <tbody>
+          {labwareByLiquidId[liquid.id].map((labware, index) => {
+            const { slotName, labwareName } = getSlotLabwareName(
+              labware.labwareId,
+              commands
+            )
+            return (
+              <TableRow
+                key={index}
+                aria-label={`LiquidDetails_${liquid.id}`}
+                onClick={() => setLabwareId(labware.labwareId)}
+              >
+                <TableDatum>
+                  <Flex>
+                    <Flex
+                      padding="0.375rem"
+                      textAlign={TYPOGRAPHY.textAlignLeft}
+                      borderRadius={BORDERS.size_three}
+                      border={`3px solid ${COLORS.darkBlackEnabled}`}
+                      fontSize={TYPOGRAPHY.fontSize20}
+                      fontWeight={700}
+                    >
+                      {slotName}
+                    </Flex>
+                  </Flex>
+                </TableDatum>
+                <TableDatum>
+                  <StyledText lineHeight={TYPOGRAPHY.lineHeight28}>
+                    {labwareName}
+                  </StyledText>
+                </TableDatum>
+
+                <TableDatum>
+                  <Flex flexDirection={DIRECTION_ROW}>
+                    <Flex
+                      height="2.75rem"
+                      padding={`${SPACING.spacing3} 0.75rem`}
+                      width="max-content"
+                      alignItems={TYPOGRAPHY.textAlignCenter}
+                      marginRight={SPACING.spacingAuto}
+                    >
+                      {getTotalVolumePerLiquidId(liquid.id, labwareByLiquidId)}{' '}
```

Comment:
need `{' '}` for space?
```suggestion
                      {getTotalVolumePerLiquidId(liquid.id, labwareByLiquidId)}{' '}
```